src/OnlineSales/Infrastructure/CustomSwaggerScheme.cs

```diff
@@ -86,8 +75,8 @@ public void Apply(OpenApiSchema schema, SchemaFilterContext context)
                             }
                             else if (property.PropertyType == typeof(DateTime) || property.PropertyType == typeof(DateTime?))
                             {
-                                SetStringExample(property, propertySchema.Value, DateTime.UtcNow.ToString("yyyy-MM-ddTHH\\:mm\\:ssZ"));
-                                propertySchema.Value.Pattern = @"^(\d{4})-(1[0-2]|[1-9])-(3[01]|[12][0-9]|[1-9])T(2[0-4]|1[0-9]|[1-9]):(2[0-4]|1[0-9]|[1-9]):([1-5]?[0-9])Z$";
+                                SetStringExample(property, propertySchema.Value, DateTime.UtcNow.ToString("O"));
```

Comment:
Can you please replace DateTime.UtcNow with a fixed date & time value. It is a bit hard to see actual changes in auto-generated swagger client when data keeps constantly changing.